jib-core/kokoro/release_build.sh

```diff
@@ -5,5 +5,5 @@ set -e
 # Display commands to stderr.
 set -x
 
-cd github/jib/jib-core
-./gradlew prepareRelease
+cd github/jib
+./gradlew :jib-core:prepareRelease
```

Comment:
I'm not actually convinced this matches up with our release scripts, I'll have to see.